applications/service_brokers/installing-service-catalog.adoc

```diff
@@ -17,3 +17,20 @@ include::modules/modules/sb-about-service-catalog.adoc[leveloffset=+1]
 
 // Installing service catalog
 include::modules/modules/sb-install-service-catalog.adoc[leveloffset=+1]
+
+// Uninstalling service catalog
+include::modules/modules/sb-uninstall-service-catalog.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+There is a known issue related to projects getting stuck in a "Terminating" state when attempting to delete them after disabling the service catalog.
+ifdef::openshift-enterprise[]
+See
```

Comment:
Needs a "the" in front of release notes -> "See the". Maybe also consider specifying the "{product-title} {product-version} Release Notes" - we only have one other link there it looks like, from the Welcome page, and that's what it uses.